railties/test/application/test_runner_test.rb

```diff
@@ -502,6 +502,14 @@ def test_pass_TEST_env_on_rake_test
       assert_match '1 runs, 1 assertions', output
     end
 
+    def test_pass_rake_options
+      create_test_file :models, 'account'
+      output =  Dir.chdir(app_path) { `bin/rake --trace=stdout test` }
+
+      assert_match '1 runs, 1 assertions', output
+      assert_match 'Execute test', output
+    end
```

Comment:
Can we add test from https://github.com/rails/rails/issues/24372 to accompany this?